README.md

```diff
@@ -22,6 +22,14 @@ You'll learn how to deploy a `cdk` template, write your first `lambda` function,
 
 By the end of the course you'll not only have a good understanding of (some of) `aws` features, you'll also have a working application deployed live on the internet.
 
+## Performance Task
```

Comment:
Maybe `assessment` or `exercise` instead of `Performance Task`?

Reply:
It’s always “exercise” or “project” or something that relates to the language of the user and not the creator.

Reply:
I think 'project' sounds best and like something I would want to do 👍 (the other sound a bit like an obligation, to me at least).